aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/request/ValidatedRequest.java

```diff
@@ -0,0 +1,28 @@
+package software.amazon.rds.dbinstance.request;
+
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class ValidatedRequest<T> extends ResourceHandlerRequest<T> {
```

Comment:
I have a concern regarding this class. If `ResourceHandlerRequest` gets updated in the wrapper package, we will have to modify this class. There is a high chance that such changes will be missed.

Reply:
`ResourceHandlerRequest` is a `@Data` class with an auto-generated builder, all-args constructor and no-args constructor. `ValidatedRequest` exposes only 1 public constructor method, which proxies to the all-args constructor. Any change to `ResourceHandlerRequest` will immediately break the all-args constructor (assuming there is any (literally any) change in the order of the arguments or their set. This is the reason I used all-args constructor: to ensure any external change would break the compilation.